programs/clearing_house/src/lib.rs

```diff
@@ -972,30 +970,34 @@ pub mod clearing_house {
             params,
         )?;
 
-        let user = &mut ctx.accounts.user;
         let order_id = {
-            let user = load!(user)?;
+            let user = load!(ctx.accounts.user)?;
```

Comment:
should this be `load_mut`?

Reply:
not sure, i guess it wouldn't compile if it's wrong

Reply:
nah because it doesn't update the user order id, just gets the last one